lib/topic_view.rb

```diff
@@ -608,7 +608,7 @@ def filtered_post_stream
       columns = [:id]
 
       if !is_mega_topic?
-        columns << 'EXTRACT(DAYS FROM CURRENT_TIMESTAMP - posts.created_at)::INT AS days_ago'
+        columns << 'ROUND(EXTRACT(EPOCH FROM CURRENT_TIMESTAMP - posts.created_at) / 86400)::INT AS days_ago'
```

Comment:
Do we actually need the `ROUND` here? I thought the `::INT` was doing it?

Reply:
I _think_ this is equivalent (but shorter)

```suggestion
        columns << '(EXTRACT(EPOCH FROM AGE(posts.created_at) / 86400)::INT AS days_ago'
```

Reply:
You are right about ROUND not being needed here. AGE subtracts from current date 12:00 AM, instead of CURRENT_TIMESTAMP.